eelbrain/_experiment/mne_experiment.py

```
@@ -4444,11 +4445,24 @@ def make_report_rois(self, test, parc=None, pmin=None, tstart=0.15, tstop=None,
         model = self._tests[test]['model']
         colors = plot.colors_for_categorial(ds.eval(model))
         test_kwargs = self._test_kwargs(samples, pmin, tstart, tstop, ('time',), None)
+        label_results = {}
+        force_permutation = (test_kwargs['samples'] and len(labels_lh + labels_rh) > 1)
```

Actually per doc string `samples` has to be `> 0` so you could just raise an error if it is not. I assume other things might break down below if it is 0.